qa/workunits/cephtool/test.sh

```diff
@@ -574,6 +574,22 @@ function test_auth_profiles()
   rm -f client.xx.keyring client.xx.keyring.2
 }
 
+function test_mon_caps()
+{
+  ./ceph-authtool --create-keyring $TMPDIR/ceph.client.bug.keyring
+  chmod +r  $TMPDIR/ceph.client.bug.keyring
+  ./ceph-authtool  $TMPDIR/ceph.client.bug.keyring -n client.radosgw.bug --gen-key
+  ./ceph -k ./keyring auth add client.radosgw.bug -i  $TMPDIR/ceph.client.bug.keyring
+  
+  if ! cat ./ceph.conf | grep client.radosgw.bug; then
+    printf "[client.radosgw.bug]\n \
+        keyring = /etc/ceph/ceph.client.bug.keyring \n" >> ./ceph.conf  
+  fi
+  ./rados lspools -n client.radosgw.bug
```

Comment:
If you want to set the keyring, you can use the --keyring argument like so:
<pre>
./rados lspools --keyring $TMPDIR/ceph.client.bug.keyring -n client.radosgw.bug
</pre>